src/pages/ticket.astro

```diff
@@ -0,0 +1,214 @@
+        elementToCapture.style.backgroundImage = "none";
+
+        // Use html2canvas to capture the content as an image
+        html2canvas(elementToCapture, { quality: 1.0 ,scale: 2,backgroundColor: "transparent"}).then(function(canvas) {
+          // Remove the background by setting the canvas background to "transparent"
+            const ctx = canvas.getContext("2d");
+            ctx.globalCompositeOperation = "destination-over";
+            ctx.fillStyle = "transparent";
+            ctx.fillRect(0, 0, canvas.width, canvas.height);
+            // Convert the canvas content to a data URL (PNG format without compression)
+            const dataURL = canvas.toDataURL("image/png");
+
+            // Create a link element for downloading the image
+            const downloadLink = document.createElement("a");
+            downloadLink.href = dataURL;
+            downloadLink.download = "VidLoveVidU2024.png";
+            document.body.appendChild(downloadLink);
+
+            // Trigger a click on the link to start the download
+            downloadLink.click();
+
+            // Remove the link from the document
+            document.body.removeChild(downloadLink);
+
+            // Restore the original background image after capturing
+            elementToCapture.style.backgroundImage = "";
+        });
+    });
+    // document.getElementById("down-ticket").addEventListener("click", function() {
+    //     const elementToCapture = document.getElementById("ticket-card");
+
+    //     // Use html2canvas to capture the content as an image with higher resolution and a transparent background
+    //     html2canvas(elementToCapture, { quality: 1.0, scale: 2, backgroundColor: "transparent" }).then(function(canvas) {
+    //         // Convert the canvas content to a data URL (JPEG format with default compression)
+    //         const dataURL = canvas.toDataURL("image/jpeg", 0.8); // 0.8 is the quality parameter (0.0 to 1.0)
+
+    //         // Open Instagram with the image ready for sharing
+    //         window.location.href = `instagram://library?AssetPath=${encodeURIComponent(dataURL)}`;
+    //     });
+    // });
+</script>
+
+<Layout title="Welcome to Astro.">
+	<main>
+		<div class="container">
+      <div class="card" id='ticket-card'>
+          <div class="username-box">
+            <p class="text-template">Username</p>
+          </div>
+          <img src="public/ดาวน์โหลด-03 5.svg" alt="" />
+          <div class="mbti-box">
+            <p class="text-template">MBTI</p>
+          </div>
+          <div class="character-box">
+            <img src="public/IMG_9059 2.png" alt="">
+          </div>
+      </div>
+      		<a class="download-box" id="down-ticket">
+       			<i class="gg-software-download"></i>
+        		<p class="text-down">Download</p>
+      		</a>
+    	</div>
+	</main>
+</Layout>
+
+<style>
+@import url('https://fonts.cdnfonts.com/css/zilla-slab');	
```

Comment:
@import statements [make websites slightly slower](https://www.debugbear.com/blog/avoid-css-import#:~:text=When%20multiple%20stylesheets%20are%20added,the%20website%20loads%20more%20slowly.). take a look if you have time (but tbh it's not too bad)